### ISSUE TITLE
Check the requirement for protoc in the ubi8-rust-builder

### ISSUE THREAD

**Issue by siegfriedweber** [stackabletech/docker-images#541]
Check the requirement for protoc in the ubi8-rust-builder

protoc was added in #167 to the ubi8-rust-builder image.

It must be checked
* if protoc is still required,
* if the hard-coded protoc version should be parameterized,
* if the protoc version should be updated,
* if the SBOM must be adapted, and
* if the upgrade of the protoc version should be added to the release steps.

**Comment by nightkr** [comment]
I'm not seeing any changes so far in https://github.com/tokio-rs/prost/releases between v0.11.0 and v0.12.3 around the dependency on protoc.

**Comment by lfrancke** [comment]
And if it is still needed it should be documented why it's added to the UBI8 builder

**Comment by nightkr** [comment]
> if the SBOM must be adapted

I'd say "yes, if we do bundle stuff like the rustc version". FWIW, I'm not seeing rustc either in https://sboms.stackable.tech/sbom/secret-operator/905b575a458feb459b3e3dbe6a2bcd70372c8054f6f0a65526b5a91a418e0799 at least. @dervoeti is this intentional?

> if the upgrade of the protoc version should be added to the release steps.

Arguably yes, but https://github.com/stackabletech/documentation/blob/314adc4e638a2f16fedf854b63f4c853080e023e/modules/ROOT/pages/release-guide.adoc doesn't seem to cover regular dependency updates either.

> if the hard-coded protoc version should be parameterized,

Only really useful IMO if we want to support multiple ubi8-rust branches in parallel.

**Comment by sbernauer** [comment]
This is `Waiting for Review`, but I'm unsure what exactly I should review. Everything said sounds correct to me.

**Comment by nightkr** [comment]
@sbernauer:
The review would be:
- Reviewing https://github.com/stackabletech/docker-images/pull/615 (already done by @lfrancke)
- Checking that my rationale for no-opping the rest makes sense (https://github.com/stackabletech/docker-images/issues/541#issuecomment-2047469199)

**Comment by sbernauer** [comment]
I think you rationales make sense in general.
However, I [added protoc to the release-bump-versions list](https://github.com/stackabletech/issues/commit/7fe9f2fa2a593353fd991b6cb06f52e07e705800)
Also, WDYT of bumping rustc `21.5` -> `26.1` right now with a PR?

**Comment by sbernauer** [comment]
Created https://github.com/stackabletech/docker-images/pull/624 for the bump

**Comment by sbernauer** [comment]
I personally would say LGTM, this can be closed
